apps/backoffice-v2/src/common/utils/svg-to-png/svg-to-png.ts

```diff
@@ -0,0 +1,23 @@
+export const svgToPng = (imageUrl: string): Promise<string> => {
```

Comment:
I think it would be better if we were to return the new URL instead of using it.

Reply:
Even if you don't persist you can pass it as an href to an img element or an anchor.